server-spi-private/src/main/java/org/keycloak/models/AdminRoles.java

```diff
@@ -69,5 +69,6 @@ public class AdminRoles {
         ALL_ROLES.add(ADMIN);
         ALL_ROLES.add(CREATE_REALM);
         ALL_ROLES.add(REALM_ADMIN);
+        ALL_ROLES.add(VIEW_SYSTEM);
```

Comment:
Is it related? This one is only used to enforce access to the server info endpoint and I guess it is working today without it here?

My concern is that this role is marked for removal in 26.4?